src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -67,6 +67,14 @@
       <Link>ExceptionStringID.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(Platform)'=='wasm'">
+    <Compile Include="..\..\Runtime.Base\src\System\Runtime\ExceptionHandling.wasm.cs">
```

Comment:
Nit: Could you please move this into the section of this file that has other Runtime.Base items?

Reply:
Done. Hopefully the failure is transient:
```
##[error]The request: 5614284 was abandoned due to an infrastructure failure. Notification of assignment to an agent was never received.
```